environments/sbox/sbox.tfvars

```diff
@@ -604,15 +604,15 @@ frontends = [
     shutter_app      = true
   },
   {
-    product          = "plum"
-    name             = "plumclassic"
-    custom_domain    = "frontdoor.sandbox.platform.hmcts.net"
-    dns_zone_name    = "frontdoor.sandbox.platform.hmcts.net"
-    backend_domain   = ["firewall-sbox-int-palo-sbox.uksouth.cloudapp.azure.com"]
-    certificate_name = "frontdoor-sandbox-platform-hmcts-net"
-    disabled_rules   = {}
-    shutter_app      = true
-    ssl_mode         = "AzureKeyVault"
+    product        = "plum"
+    name           = "plumclassic"
+    custom_domain  = "hmcts-sbox.azurefd.net"
```

Comment:
err this won't work

Reply:
?

Reply:
the custom domain needs to be one you want not the cname target.

unsure why it didn't validate before but maybe DNS hadn't caught up, it looks fine now:

```
❯ dig frontdoor.sandbox.platform.hmcts.net

; <<>> DiG 9.10.6 <<>> frontdoor.sandbox.platform.hmcts.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 2622
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;frontdoor.sandbox.platform.hmcts.net. IN A

;; ANSWER SECTION:
frontdoor.sandbox.platform.hmcts.net. 60 IN A	13.107.246.64
frontdoor.sandbox.platform.hmcts.net. 60 IN A	13.107.213.64

;; Query time: 96 msec
;; SERVER: 100.64.9.0#53(100.64.9.0)
;; WHEN: Fri Sep 29 15:58:23 BST 2023
;; MSG SIZE  rcvd: 97


☸ cft-preview-01-aks-admin (et) in rpx-xui-webapp on  try-faster-build [$!] via 🐳 desktop-linux via  v14.21.3
❯ dig hmcts-sbox.azurefd.net

; <<>> DiG 9.10.6 <<>> hmcts-sbox.azurefd.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 52926
;; flags: qr rd ra; QUERY: 1, ANSWER: 5, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;hmcts-sbox.azurefd.net.		IN	A

;; ANSWER SECTION:
hmcts-sbox.azurefd.net.	30	IN	CNAME	star-azurefd-prod.trafficmanager.net.
star-azurefd-prod.trafficmanager.net. 40 IN CNAME dual.part-0036.t-0009.t-msedge.net.
dual.part-0036.t-0009.t-msedge.net. 42 IN CNAME	part-0036.t-0009.t-msedge.net.
part-0036.t-0009.t-msedge.net. 42 IN	A	13.107.246.64
part-0036.t-0009.t-msedge.net. 42 IN	A	13.107.213.64

;; Query time: 95 msec
;; SERVER: 100.64.9.0#53(100.64.9.0)
;; WHEN: Fri Sep 29 15:58:33 BST 2023
;; MSG SIZE  rcvd: 189

```

Reply:
sure - let me re-run pipeline from `master` to see how it goes

Reply:
hmm - still failing, will give some time and try again.  Although I think I have done the DNS change about hour ago so thought it would be enough..
https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=446412&view=logs&j=b0a0f563-df07-5ab0-43a3-a747dcffaf72&t=6bda5314-df61-500a-4862-86324a7e7d94